pkg/condition/parameters.go

```diff
@@ -0,0 +1,47 @@
+package condition
+
+import (
+	"fmt"
+
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"github.com/openfga/openfga/pkg/condition/types"
+)
+
+// castContextToTypedParameters takes a map of context and a map of condition parameter
+// types and casts/converts the map into a map of typed values that can be evaluated
+// with strong type guarantees.
+func castContextToTypedParameters(
+	contextMap map[string]any,
+	conditionParameterTypes map[string]*openfgav1.ConditionParamTypeRef,
+) (map[string]any, error) {
+	if len(contextMap) == 0 {
```

Comment:
Can we add a test here and below to get coverage on these lines? I realize it's a bit overkill, but may as well make Codecov happy.